docs/Community/sig-attestation/README.md

```diff
@@ -0,0 +1,26 @@
+Attestation Special Interest Group
+==================================
+
+SIG-Attestation is responsible for code, plugins, testing, and other works
+related to or required for in-enclave attestation and integration with
```

Comment:
remove "in-enclave" since there is an experimental "host verification" library for verifying attestation reports outside of an enclave.

Reply:
Oh, neat!